tests/integration/ar-io-writable.test.ts

```diff
@@ -132,12 +136,34 @@ describe('ArIOWriteable', () => {
   it('should successfully increase the undername support on a domain', async () => {
     const domain = 'test-record';
     const qty = 1;
-    const tx = await arIO.increaseUndernameLimit({
-      domain,
-      qty,
-    });
-    expect(tx.id).toBeDefined();
-    const verified = await arweave.transactions.verify(tx as Transaction);
+    const res = await arIO.increaseUndernameLimit(
+      {
+        domain,
+        qty,
+      },
+      {
+        tags: [
+          {
+            name: 'ar-io-sdk-write',
+            value: AR_IO_CONTRACT_FUNCTIONS.INCREASE_UNDERNAME_LIMIT,
+          },
+        ],
+      },
+    );
+
+    const tx = await arweave.transactions.get(await res.id);
```

Comment:
```suggestion
    const tx = await arweave.transactions.get(res.id);
```

Reply:
for some reason res.id can be a promise or a string

Reply:
Yeah because we return Transaction OR DataItem, we are chained to this type 
![image](https://github.com/ar-io/ar-io-sdk/assets/85306700/2c2250a1-0683-42c0-afdd-9a14307fdf27)
I might be able to add an overload to fix this